src/style.css

```
@@ -0,0 +1,46 @@
+
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.box {
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  width: 96px;
+}
+
+.stars {
+  box-sizing: border-box;
+  display: flex;
+  justify-content: space-between;
+  height: 16px;
+}
+
+.stars__star {
+  padding: 0 1.5px;
+  width: 13px;
+  content: url(images/star.svg);
+}
+
+.stars--1 span:first-child {
```

```suggestion
.stars--1 .stars__star:first-child {
```
У этого элемента есть класс